### DISCUSSION TITLE
ConfigureAwait(false)

### DISCUSSION THREAD

**Discussion by kindermannhubert** [waf/PrettyPrompt#70] [General]
We have 15 places in PrettyPrompt where we await async method with ConfigureAwait(false) and 12 places where we await async method without it.
Is this intentional or is it possible to say that all awaits in PrettyPrompt should/could have ConfigureAwait(false)?

If ConfigureAwait(false) should be used everywhere I would enable [CA2007](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2007) and add missing calls. Writing those calls is extremely easy to forget about.

@waf could you advise, please?

**Comment by waf** [comment]
The intention was to `ConfigureAwait(false)` everywhere -- the awaits missing it should be corrected.

Enabling CA2007 is a great idea

**Reply by kindermannhubert** [reply]
Done.
